assets/js/pages/HostDetailsPage/HostDetails.jsx

```diff
@@ -117,6 +117,7 @@ function HostDetails({
   const saptuneVersion = get(saptuneStatus, 'package_version');
   const saptuneConfiguredVersion = get(saptuneStatus, 'configured_version');
   const saptuneTuning = get(saptuneStatus, 'tuning_state');
+  const currentlyAppliedSolution = get(saptuneStatus, 'enabled_solution.id');
```

Comment:
I am wondering whether we should use `applied_solution` here

Reply:
Yes, maybe it would be better. But I think it will be the same always